next.js/packages/one/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+
+npm_link_all_packages(name = "node_modules")
+
+ts_project(
+    name = "one_ts",
+    srcs = ["src/main.ts"],
+    allow_js = True,
+    # runtime direct dependencies of the linked npm package
+    data = [
+        "//packages/one:node_modules/is-odd",
```

Comment:
nit: why not just ":node_modules/is-odd"

fully qualifying seems like it makes the example look more complex

Reply:
🦕